driver/driver_test.go

```diff
@@ -30,3 +32,49 @@ func TestDebugDriver_Run(t *testing.T) {
 	}
 	is.NoError(d.Run(op))
 }
+
+func TestOperation_Unmarshall(t *testing.T) {
+	var op Operation
+	is := assert.New(t)
+	bytes, err := ioutil.ReadFile("../testdata/operations/valid-operation.json")
+	is.NoError(err, "Error reading from testdata/operations/valid-operation.json")
+	is.NoError(json.Unmarshal(bytes, &op), "Error unmarshalling operation")
+	is.NotNil(op, "Expected Operation not to be nil")
+}
+
+func TestOperation_Marshall(t *testing.T) {
+
+	actualOp := Operation{
+		Action:       "install",
+		Installation: "test",
+		Parameters: map[string]interface{}{
+			"param1": "value1",
+			"param2": "value2",
+		},
+		Image:     "testing.azurecr.io/duffle/test:e8966c3c153a525775cbcddd46f778bed25650b4",
+		ImageType: "docker",
+		Revision:  "01DDY0MT808KX0GGZ6SMXN4TW",
+		Environment: map[string]string{
+			"ENV1": "value1",
+			"ENV2": "value2",
+		},
+		Files: map[string]string{
+			"/cnab/app/image-map.json": "{}",
+		},
+		Out: os.Stdout,
+	}
+	is := assert.New(t)
+	bytes, err := json.Marshal(actualOp)
+	is.NoError(err, "Error Marshalling actual operation to json")
+	is.NotNil(bytes, "Expected marshalled json not to be nil")
+	actualJSON := string(bytes)
+	var expectedOp Operation
+	bytes, err = ioutil.ReadFile("../testdata/operations/valid-operation.json")
```

Comment:
Why don't you just check the equality between those bytes and the actualJSON bytes?

Reply:
I did it this way so that I don't have to worry about the formatting of the json, if the way that operation is serialized changes then this will still work. I guess I could have minified the json and compared those bytes but this didn't require any additional libs.

Reply:
Speaking as someone who just had to work with a test in cnab-go that asserted on the bytes instead of the json string representation, comparing strings instead of bytes is also a lot easier to deal with when the test fails or needs to be updated.